templates/azure-functions/.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
We could lower this to v14 I think?

```suggestion
14
```

This will make it consistent with all other minimum needed Node versions.